pkg/buildkite/pipelines.go

```diff
@@ -282,6 +283,7 @@ type CreatePipelineArgs struct {
 	SkipQueuedBranchBuilds    bool     `json:"skip_queued_branch_builds"`
 	CancelRunningBranchBuilds bool     `json:"cancel_running_branch_builds"`
 	Tags                      []string `json:"tags"`
+	CreateWebhook             *bool    `json:"create_webhook"`
```

Comment:
Just make `CreateWebhook` a bool, the MCP library will default it to true.

Reply:
@wolfeidau Yeah, just re-confirmed: after reverting [7357483](https://github.com/buildkite/buildkite-mcp-server/pull/160/commits/73574834aa896dbae347977215fe79c225722f16) to use `bool` rather than `*bool`, the tool gets called with `args.CreateWebhook` set to `false` when the prompt doesn't contain an explicit request to create the webhook. 🤷🏻 

I'll share a quick video showing both cases end to end.

Reply:
Here you go: https://cnunciato-shared.s3.us-east-1.amazonaws.com/mcp-server-create-webhook-example.mp4